rotkehlchen/db/evmtx.py

```diff
@@ -508,6 +519,11 @@ def delete_transactions(
             'DELETE FROM evm_tx_mappings WHERE tx_id=? AND value=?',
             [(x, HISTORY_MAPPING_STATE_DECODED) for x in tx_ids],
         )
+        # Delete any key_value_cache entries
+        write_cursor.executemany(
+            'DELETE FROM key_value_cache WHERE name LIKE ?',
+            [(f'extrainternaltx_{tx_hash.hex()}%',) for tx_hash in tx_hashes],
```

Comment:
make this into a constant final variable

```suggestion
            [(f'{EXTRAINTERNALTXPREFIX}_{tx_hash.hex()}%',) for tx_hash in tx_hashes],
```